### PR TITLE
feat: New Module `avm/res/portal/dashboard`

### PR DESCRIPTION
**Pull request by krbar**
## Description

New Module `avm/res/portal/dashboard`.

Related Module Proposal: https://github.com/Azure/Azure-Verified-Modules/issues/980

## Adding a new module

- [x] A proposal has been submitted and approved.
- [ ] I have included "Closes #{module_proposal_issue_number}" in the PR description.
- [ ] I have run `brm validate` locally to verify the module files.
- [x] I have run deployment tests locally to ensure the module is deployable.

## Pipeline references

| Pipeline |
| - |
| [avm.res.portal.dashboard](https://github.com/krbar/bicep-registry-modules/actions/workflows/avm.res.portal.dashboard.yml) |

### CODE REVIEW

**Comment by krbar** [comment]
Converted the PR to Draft until it appears in the index and passes the static validation.

**Comment by krbar** [comment]
@Azure/avm-core-team-technical-bicep can I kindly ask to review this PR?

**Review by jtracey93 — APPROVED** [review]
LGTM 🚀

File: avm/res/portal/dashboard/main.bicep [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+    'Microsoft.Authorization/roleDefinitions',
+    '18d7d88d-d35e-4fb5-a5c3-7773c20a72d9'
+  )
+}
+
+// ============== //
+// Resources      //
+// ============== //
+
+resource avmTelemetry 'Microsoft.Resources/deployments@2023-07-01' = if (enableTelemetry) {
```
Comment (AlexanderSehr) [inline avm/res/portal/dashboard/main.bicep:48]: Should've had the suppression flag added

**Pull request merged**